### PR TITLE
Updated documentation on how to update documentation

### PR DESCRIPTION
**Pull request by samuelshaner**
This PR updates the documentation on how to update documentation based on the new documentation workflow.

### CODE REVIEW

File: docs/source/devguide/documentation.rst [hunk 1]
```diff
@@ -64,87 +31,77 @@ Writing New Documentation
...
+The reStructuredText source files that should be updated with each pull request are located in the :file:`/docs/source` directory. If developers wish to add new pictures to the documentation, image files should be added to the :file:`/docs/img` directory in either png or svg format. Developers can test out their changes to the documentation by recompiling the Sphinx documentation into HTML and viewing the updated HTML files.
 
-In order to **expose** your newly generated HTML documentation to GitHub such that it will present your website, you must copy the HTML into the main :file:`sphinx` directory:
+Likewise, in order to build a revised version of the :ref:`OpenMOC API <api>` from Doxygen-style comments in the Python and C++ source code, you may use the following console command from within the :file:`docs/doxygen` directory:
 
 .. code-block:: none
 
-    cd sphinx
-    cp -rf docs/build/html/. .
+    cd doxygen
```
Comment (wbinventor) [inline docs/source/devguide/documentation.rst]: Does this mean the user will be in the "OpenMOC/docs/doxygen/doxygen" or "OpenMOC/docs/doxygen" directory?

Comment (samuelshaner) [inline docs/source/devguide/documentation.rst]: good catch - the user should be in `OpenMOC/docs/doxygen`

File: docs/source/devguide/documentation.rst [hunk 2]
```diff
@@ -64,87 +31,77 @@ Writing New Documentation
...
 
+Since the :file:`.gitignore` file does not allow image and HTML files to be tracked, you will need to force all image and HTML files to be added to tracking history. The :file:`configure.sh` script contains the commands to add all the necessary files. The :file:`configure.sh` can be executed and the changes committed and pushed to GitHub:
 
+.. code-block:: none
 
+    ./configure.sh
+    git commit -am 'my commit message here'
+    git push origin gh-pages-update
 
+The final step in updating the website documentation is to `pull request`_ to the ``gh-pages`` branch of the OpenMOC repository on GitHub as described in :ref:`Development Workflow <work_flow>`.
```
Comment (wbinventor) [inline docs/source/devguide/documentation.rst]: Perhaps note that the only reason we want PRs to "gh-pages" is to provide a transparent and publicly-available record of when the docs on the web have changed - and *not* to review the diff!

**Comment by wbinventor** [comment]
This looks good to me!

**Comment by samuelshaner** [comment]
I've updated the docs based on the comments.

**Pull request merged**